.github/workflows/deploy.yaml

```
@@ -0,0 +1,51 @@
+name: Build and Deploy to Netlify
+
+on:
+  push:
+    branches:
+      - main
+      - master
```

We'll only use `main` in this repository:

```suggestion
      - main
```